plugins/action/tests/integration/ndfc_network_validate.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+from pprint import pprint
+import json
+from ansible.utils.display import Display
+from ansible.plugins.action import ActionBase
+from ansible.module_utils.six import raise_from
+from ansible.errors import AnsibleError
+from ansible.module_utils.common.text.converters import to_native
+from ..plugin_utils.tools import load_yaml_file, process_deepdiff
+from ..plugin_utils.pydantic_schemas.dcnm_network.schemas import DcnmNetworkQuerySchema
+
+try:
+    from deepdiff import DeepDiff
+except ImportError as imp_exc:
+    DEEPDIFF_IMPORT_ERROR = imp_exc
+else:
+    DEEPDIFF_IMPORT_ERROR = None
+
+if DEEPDIFF_IMPORT_ERROR:
+    raise_from(
+        AnsibleError('DeepDiff must be installed to use this plugin. Use pip or install test-requirements.'),
+        DEEPDIFF_IMPORT_ERROR)
+
+display = Display()
+
+
+class ActionModule(ActionBase):
+
+    def run(self, tmp=None, task_vars=None):
+        results = super(ActionModule, self).run(tmp, task_vars)
+        results['failed'] = False
+
+        if 'ndfc_data' not in self._task.args:
```

Comment:
Suggest you do this in a loop and check for all action plugin input params.

```python
for input_item in ['ndfc_data', 'test_data', 'config_path']:
    if input_item not in self._task.args:
        results['failed'] = True
        results['msg'] = f"Required input parameter not found: '{input_item}'"
        return results
```

Fair warning... I did not test this code :)

Reply:
Then just set all 3 at once after the loop/check

```python
ndfc_data = self._task.args['ndfc_data']
test_data = self._task.args['test_data']
config_path = self._task.args['config_path']
```